src/TrackRowInfoVisBar.js

```diff
@@ -36,7 +36,7 @@ export default function TrackRowInfoVisBar(props) {
         viewId,
         trackId,
         transformedRowInfo,
-        sortOrder,
+        titleSuffix,
```

Comment:
Can you update the prop comments to reflect these prop name changes?

Reply:
Of course